front/app/modules/commercial/content_builder/admin/components/ContentBuilderLayout/index.tsx

```diff
@@ -27,7 +62,22 @@ const ContentBuilderLayout: React.FC<ContentBuilderLayoutProps> = ({
     return null;
   }
 
-  return <>{children}</>;
+  return (
+    <Editor resolver={{ Box, Container }} onRender={RenderNode}>
+      <Wrapper>
+        <Box zIndex="100" margin-left="-100px" w="100%" position="fixed">
```

Comment:
The reason this is misbehaving is the `position: fixed` on the left column - maybe try removing it and creating the layout without the need for these props?